tests/system/small/ml/test_preprocessing.py

```diff
@@ -244,6 +244,8 @@ def test_min_max_scaler_series_normalizes(penguins_df_default_index, new_penguin
     # If minmax-scaled correctly, min should be 0 and max should be 1.
     for column in result.columns:
         assert math.isclose(result[column].max(), 1.0, abs_tol=1e-3)
+    for column in result.columns:
```

Comment:
nit: you can do both checks in the same loop.

Reply:
Done